src/frontend/src/eth/stores/fee.store.ts

```diff
@@ -25,6 +26,7 @@ export const initFeeStore = (): FeeStore => {
 export interface FeeContext {
 	feeStore: FeeStore;
 	feeSymbolStore: Writable<string | undefined>;
+	feeIdStore: Writable<TokenId | undefined>;
```

Comment:
- What's a `feeId`, ID of what?

- Maybe not for this PR but, given that now two information derives from the same source we should probably pass a single source to the context and derives the symbol and id as we do in the `SendContext`

Reply:
- changing to `feeTokenId`

- agreed! but I was waiting to have the balance in the tokens too, since we use it somewhere